### PR TITLE
Update CoAp server + readme

### PR DESCRIPTION
**Pull request by matluuk**
Major update to CoAp server and readme.

Some of the changes:
- update server to get ip and port from command line erguments.
- Remove unnecessary resources from the server
- Add guide to readme, how to deploy the server using google cloud VM
-  Fixes to server logging

### CODE REVIEW

**Review by toukokinnunen — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -215,9 +182,243 @@ Set up Fit IoT-Lab experiment with two nodes for device firmware and border rout
...
+
+### Set up Google cloud VM
+
+Google Cloud account is needed for using google Cloud services. At the time being, 300 credits can be obtained for use in 3 months time period for free. Below are a link to where a google cloud account can be created.
+
+https://console.cloud.google.com/?hl=en&_ga=2.87814171.-2055644655.1699615458&_gac=1.116033140.1702051941.CjwKCAiAmsurBhBvEiwA6e-WPIQOg4lsX1QJevny4vxo9FBotFtCxOCFgTHR5MXrhSOSkf66HEamdRoCthsQAvD_BwE
+
+### Create Virtual private cloud
+
+Next step is to create vpc (virtual private cloud) for the VM. This is necessary to bget the external ipv6 address for the VM. The VPC:s can be managed at following google cloud console site.
```
Comment (HHanhela) [inline README.md:204]: Typo, bget -> get

File: README.md [hunk 2]
```diff
@@ -215,9 +182,243 @@ Set up Fit IoT-Lab experiment with two nodes for device firmware and border rout
...
+
+<details><summary>Firewall rule: allow udp 8683</summary>
+
+![Firewall rule CoAp 1](images/firewallCoap1.png)
+
+![Firewall rule CoAp 2](images/firewallCoap2.png)
+
+</details>
+
+If you have followd according to this tutorial, you should have these firewall rules for the newly created VPC. Notice, wtät there are also some firewall rules for default VPC.
```
Comment (HHanhela) [inline README.md:249]: Typo, wtät -> that

**Review by HHanhela — CHANGES_REQUESTED** [review]
Fix typos.

File: README.md (hunk 1 above)
Comment (matluuk) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (matluuk) [reply README.md]: Fixed

**Pull request merged**